armi/reactor/grids.py

```diff
@@ -967,17 +1054,65 @@ def changePitch(self, xw, yw):
         newOffsetY = self._offset[1] * yw / ywOld
         self._offset = numpy.array((newOffsetX, newOffsetY, 0.0))
 
-    @property
-    def pitch(self):
-        """
-        The pitch of the grid.
+    def getSymmetricEquivalents(self, indices):
```

Comment:
same about docstring. I'm ok with this if the parent has a good one.

Reply:
sphinx is re-using the base class docstring